deploy/helm/sumologic/README.md

```diff
@@ -27,7 +27,7 @@ The following table lists the configurable parameters of the Sumo Logic chart an
 | `sumologic.cleanupEnabled`                                                       | If enabled, a pre-delete hook will destroy Kubernetes secret and Sumo Logic Collector.                                                                                                                                                                                                                                                                               | `false`                                                                                                                                                                              |
 | `sumologic.events.enabled`                                                       | Defines whether collection of Kubernetes events is enabled.                                                                                                                                                                                                                                                                                                          | `true`                                                                                                                                                                               |
-| `sumologic.events.provider`                                                      | Defines which provider is used for Kubernetes events collection. This can be either `fluentd` or `otelcol`.                                                                                                                                                                                                                                                          | `otelcol`                                                                                                                                                                            |
+| `sumologic.events.provider`                                                      | Defines which provider is used for Kubernetes events collection - `otelcol` or `fluentd`. Otelcol is the default and is recommended. Fluentd is deprecated.                                                                                                                                                                                                          | `otelcol`                                                                                                                                                                            |
```

Comment:
I suspect this configuration is case sensitive

```suggestion
| `sumologic.events.provider`                                                      | Defines which provider is used for Kubernetes events collection - `otelcol` or `fluentd`. `otelcol` is the default and is recommended. `fluentd` is deprecated.                                                                                                                                                                                                      | `otelcol`                                                                                                                                                                            |
```

Reply:
Changed